Makefile

```diff
@@ -60,6 +60,11 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+BUILDER_IMAGE_PRIVATE = registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11
+BUILDER_IMAGE_PUBLIC  = registry.ci.openshift.org/openshift/release:golang-1.18
+TARGET_IMAGE_PRIVATE  = registry.ci.openshift.org/ocp/4.11:base
+TARGET_IMAGE_PUBLIC   = registry.ci.openshift.org/origin/4.10:base
+
```

Comment:
Can you please change the assignments to use `?=`. This will allows us to set these externally if required.
Also do you mind adding few lines in the doc to use the public targets .

Reply:
Sure!

Reply:
Fixed.